src/index.js

```diff
@@ -149,7 +149,9 @@ export function createComplexityLimitRule(maxCost, options = {}) {
         }
 
         if (node.kind === 'Document') {
-          if (visitor.getCost() > maxCost) {
+          const cost = visitor.getCost();
+          if (options.onCost) options.onCost(cost);
```

Comment:
I still think passing the document node might be useful. we maybe want to log queries that fail in prod as well?